pages/auth-pages/login.html

```diff
@@ -0,0 +1,143 @@
+                </a>
+            </div>
+            <!-- logo -->
+
+            <!-- searchbar -->
+            <form class="search__bar__container ">
+                <input type="text" class="search__bar bg-grey-700" placeholder="Search">
+                <a href="#"><i class="fas fa-search search__icon text-fuchsia-500"></i></a>
+            </form>
+            <!-- searchbar -->
+
+
+            <!-- icons (profile) -->
+            <div class="flex justify-center align-items-center">
+
+                <a href="">
+                    <button style="background: transparent;"
+                        class="btn flex flex-col text-fuchsia-500 text-hover-fuchsia-400 tooltip">
+                        <span><i class="text-lg fas fa-user "></i> </span>
+
+                        <span style="text-transform: none;" class="tooltip-text">Profile</span>
+
+                    </button>
+                </a>
+
+                <a href="/pages/auth-pages/login.html" class="btn btn-solid-fuchsia text-white ml-5">Login</a>
+            </div>
+            <!-- icons (profile) -->
+        </div>
+    </header>
+    <!-- header -->
+
+    <!-- main -->
+    <main style="top: 5rem; min-height: 100vh" class="relative w-full  flex justify-center align-items-center">
+        <div style="margin: auto; " class="h=96 w-96 p-10 shadow-2xl rounded-2xl">
+
+            <h3 class="text-2xl font-normal text-center text-black mb-20">Login</h3>
+
+
+            <div class="mb-5">
+                <div class="input-container text-black">
+                    <input type="text" id="email" class="input-outlined input-outlined-fuchsia" autocomplete="off"
+                        placeholder=" " />
+                    <label style="background-color: white" for="email" class="input-label ">Email *</label>
+                </div>
+            </div>
+
+
+            <div class="mb-5">
+                <div class="input-container text-black">
+                    <input type="text" id="email" class="input-outlined input-outlined-fuchsia" autocomplete="off"
+                        placeholder=" " />
+                    <label style="background-color: white;" for="email" class="input-label ">Password *</label>
+                </div>
+            </div>
+
+            <div class="flex justify-between mb-10">
+                <div class="flex justify-center align-items-center">
+                    <input type="checkbox" class="mr-3" name="" id="">
+                    <h6 class="text-lg text-black font-normal">Remember me</h6>
+                </div>
+                <a href="/pages/auth-pages/signup.html" class="btn btn-link-fuchsia text-black">Sign Up ?</a>
+            </div>
+
+            <div>
+                <a href="" class="btn btn-link-fuchsia text-black">Forgot Password ?</a>
+
+            </div>
+        </div>
+    </main>
+    <!-- main -->
+
+    <!-- footer -->
+    <footer style="bottom: 0"
```

Comment:
I'm using inline styles here because I want to overwrite a few properties that the classes from my component library are applying and because inline styles are more specific and will overwrite im using them in a few places